basxconnect/core/layouts/editlegalperson.py

```diff
@@ -41,7 +41,7 @@ def base_data_tab():
                         ),
                     ),
                 ),
-                gridmode="full-width",
+                gutter=False,
```

Comment:
Would it make sense to set ```gutter=False``` as a default in the Grid-element? Or this more of a special case because of the complex person-edit screen?

Reply:
I used it here because this is what they did in the carbon design react tutorial when they nested grids but wanted the inner grid to have the same width as the outer one.

Reply:
Ok, I see